src/test/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessMessageServiceTest.java

```diff
@@ -0,0 +1,276 @@
+package uk.gov.hmcts.reform.sscs.service.hmc.topic;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
+import uk.gov.hmcts.reform.sscs.exception.*;
+import uk.gov.hmcts.reform.sscs.model.hmc.message.HearingUpdate;
+import uk.gov.hmcts.reform.sscs.model.hmc.message.HmcMessage;
+import uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingGetResponse;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingResponse;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.RequestDetails;
+import uk.gov.hmcts.reform.sscs.service.CcdCaseService;
+import uk.gov.hmcts.reform.sscs.service.HmcHearingApiService;
+
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.State.UNKNOWN;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.CANCELLED;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.EXCEPTION;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.HmcStatus.LISTED;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.ListingStatus.DRAFT;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.ListingStatus.FIXED;
+
+@ExtendWith(MockitoExtension.class)
+class ProcessMessageServiceTest {
+
+    public static final String HEARING_ID = "abcdef";
+    public static final long CASE_ID = 123L;
+
+    private SscsCaseDetails sscsCaseDetails;
```

Comment:
very minor: need some consistency where we put these, I would prefer after the unitUnderTest like you've done for the others.